tests/v1/lifecycle/test_systems.py

```diff
@@ -51,4 +53,4 @@
 def test_system_specified(client, api_prefix, source_data, path, response, monkeypatch):
     monkeypatch.setattr(roadmap.v1.lifecycle.systems, "OS_DATA_MOCKED", source_data)
     data = client.get(f"{api_prefix}/lifecycle/systems{path}")
-    assert data.json() == response
+    assert data.json() == sorted(response, key=itemgetter("major", "minor"), reverse=True)
```

Comment:
@samdoran Just thing to consider I found when I was changing the tests:

I was also about using the `sorted` for the response. But I was thinking that the code in the unit tests and in the code would be mostly simmilar and if there would be some mistake in the algorithm (the chance is low there, but we never know) we won't find it. So I rather changed the order of the expected order to have exactly what we want and don't touch the data in the test. I used for that some "reverse order of lines" tool, so it was pretty quick.